examples/quantization_non_uniform/run_gptq_nvfp4_multi_model_evaluation.sh

```
@@ -0,0 +1,440 @@
+print("Key:")
+print("  ✓ = Improved (better with round_scales=True)")
+print("  ✗ = Degraded (worse with round_scales=True)")
+print("  ↑ = Value increased")
+print("  ↓ = Value decreased")
+print("=" * 80)
+print()
+
+EOF
+
+# Export variables for Python script
+export MODEL_NAMES="${MODEL_NAMES[*]}"
```

Shellcheck (0.11.0)

[warning] Variable was used as an array but is now assigned a string.

(SC2178)